packages/Popover/stories/examples/ScrollContainer.js

```diff
@@ -62,15 +65,52 @@ PopoverBox.propTypes = {
   getScrollContainer: PropTypes.func.isRequired,
 };
 
-const ExampleStory = props => (
-  <React.Fragment>
-    <CenteredStory>
-      <ScrollBox id="scroll-container">
-        <PopoverBox {...props} getScrollContainer={() => document.querySelector("#scroll-container")} />
-      </ScrollBox>
-    </CenteredStory>
-    <TrailerBox>Fin.</TrailerBox>
-  </React.Fragment>
-);
+function setKnobsTab() {
```

Comment:
Since this story has no "actions", it will always reset the addon tab to the "knobs".